apps/web/pages/api/book/event.ts

```diff
@@ -335,7 +335,17 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   const teamMembers = await Promise.all(teamMemberPromises);
 
-  const attendeesList = [...invitee, ...guests, ...teamMembers];
+  const theOrganizer = {
+    email: users[0].email,
+    name: users[0].name,
```

Comment:
We're failing on E2E tests here due to name and being possible null 

( I tried making this change myself but don't appear to have permission to push to this branch 🤔)

```suggestion
    name: users[0].name || "",
```

Reply:
Yeah, it's in our fork.  I'll fix it.